diff --git a/Makefile.am b/Makefile.am
index 7c2b006..1a3eb53 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,5 +1,4 @@
 bin_PROGRAMS = xte rgb2pat png2pat visgrep pat2ppm patextract xmousepos
-man_MANS = pat2ppm.1 patextract.1 png2pat.1 rgb2pat.1 visgrep.1 xautomation.7 xmousepos.1 xte.1
 xte_SOURCES = xte.c debug.c kbd.h keysym_map.h multikey_map.h
 xte_LDADD = $(X_LIBS) $(X_EXTRALIBS) -lXtst -lX11
 rgb2pat_SOURCES = rgb2pat.c image.c image.h debug.c debug.h
